tests/contracts/root_chain/test_start_standard_exit.py

```diff
@@ -221,6 +226,7 @@ def test_start_standard_exit_from_two_deposits_with_the_same_amount_and_owner_sh
     testlang.start_standard_exit(second_deposit_id, owner)
 
 
+@pytest.mark.skip("Fails due to being non-conformant to ERC712")
```

Comment:
ERC -> EIP?

Reply:
I think the reason is not EIP712 but rather that this test calls challenge function.